### ISSUE TITLE
bur:12994

### ISSUE THREAD

**Issue by funderburkjim** [sanskrit-lexicon/csl-orig#448]
date:	01/04/2021 16:45:28
dict:	bur
Lnum:	12994
hw:	madhu
## old:
Hello, several, 
 first notice that the abreviation "bot." (many occurrences here) doesn't work (a problem i guess related to the fact it is here at the end of sentences); 

note also "esp. de de datte." (a misprint in the book) should be "esp. de datte." 

Also "c. à d." (several occurences here, and probably in the whole dictionary i guess) should be "c.-à-d." (i.e.) also one occurrence of "c. a d." (not properly referenced and written), same, to be replaced by "c.-à-d." 

also two occurences of "sansevière" (a plant name) mispelled (sanseviére & sanseviere).  LB 

Also Burnouf on this link "https://www.sanskrit-lexicon.uni-koeln.de/scans/awork/apidev/simple-search/v1.0/list-0.2s.html" is one of the last dictionaries of Koel univ which still display nicely the text in grey without adding on top a grey highlighting (not to speak of the big highlighting on the whole line in MM instead of the previous discreet one)

## new:
	
several, first notice that the abbreviation "bot." (many occurrences here) doesn't work (a problem i guess related to the fact it is here at the end of sentences); 

note also "esp. de de datte." (a misprint in the book) should be "esp. de datte." Also "c. à d." (several occurences here, and probably in the whole dictionary i guess) should be "c.-à-d." (i.e.) also one occurrence of "c. a d." (not properly referenced and written), same, to be replaced by "c.-à-d." 

also two occurences of "sansevière" (a plant name) mispelt (sanseviére & sanseviere).  LB 

Also Burnouf on this link "https://www.sanskrit-lexicon.uni-koeln.de/scans/awork/apidev/simple-search/v1.0/list-0.2s.html" is one of the last dictionaries of Koel univ which still display nicely the text in grey without adding on top a grey highlighting (not to speak of the big highlighting on the whole line in MM instead of the previous discreet one)
comm:	Typo, etc.

**Comment by funderburkjim** [comment]
## bot abbreviation

This due to an error in burab_input.txt in csl-pywork.
Now corrected

**Comment by funderburkjim** [comment]
esp. de de datte -> esp. de datte

Also two others noticed and corrected:
```
18737  : subAhu   : fils de de Dhṛtarāṣṭra : fils de Dhṛtarāṣṭra :
19541  : havis    : mêlé de de beurre : mêlé de beurre : 
```

marked all 3 as print changes.

**Comment by funderburkjim** [comment]
c-à-d.

The most common in bur (according to our digitization) is `c-à-d.` (191 lines).  (no period after 'c')

Next most common is `c. à d.` (46 lines) (period after 'c')

Google translate recognizes `c-à-d.` as 'ie.'  and `c.-à-d.` as 'i.e.'.

But `c. à d.` is recognized as 'vs. to d.'  which looks wrong.

Will change `c. à d.`  to `c.-à-d.`  and note same in print_change .

**Comment by funderburkjim** [comment]
sansevière  occurs 8 times in our digitization

sanseviére and sanseviere occur only once, in maDu. And the print is smudged in both cases regarding
the 'er'.   Changing both to use `ère ` and considering this a typo, not a print change.

Incidentally, there are also 4 occurrences of `sanseviera`  (no accent, end with 'a') always as
`sanseviera zeylanica`.  No change made in these.

**Comment by funderburkjim** [comment]
## display concerns

Not sure what this means.  Will enter as separate issue

**Comment by gasyoun** [comment]
@funderburkjim seems to contain millions of errors. And the same user submits the mistakes, right? 
I would go for batch corrections, otherwise it is too time consuming for a 2-nd priority small dictionary.

**Comment by funderburkjim** [comment]
Caujolle has added a lot over the years to the quality of the French dictionaries, especially.
I feel obligated to take her concerns seriously.
